core/src/main/res/values/strings.xml

```diff
@@ -779,8 +779,7 @@
     <!-- Share episode dialog -->
     <string name="share_dialog_include_label">Include:</string>
     <string name="share_playback_position_dialog_label">Playback position</string>
-    <string name="share_dialog_media_file_url_label">Media file address</string>
-    <string name="share_dialog_episode_website_label">Episode webpage</string>
+    <string name="share_episode_website_and_file_url_label">Episode webpage and Media file address</string>
```

Comment:
This is a pretty technical explanation. How about just calling it something like "Link to episode"? @keunes

Reply:
Yep, sounds better :)